packages/meditrak-server/src/apiV2/import/importSurveyResponses/assertCanImportSurveyResponses.js

```diff
@@ -31,6 +31,9 @@ export const assertCanImportSurveyResponses = async (
       'code',
       surveyResponseCountryCodes,
     );
+    if (surveyResponseCountries.length !== surveyResponseCountryCodes.length) {
```

Comment:
@IgorNadj added this check because previously "Demo Land" didn't exist in tests, so they were (wrongly) making it through the permissions check